evaluation/mle_bench/environment/Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM ubuntu:20.04
+
+# Avoid interactive dialog from apt-get and other packages requiring configuration
+ENV DEBIAN_FRONTEND=noninteractive
+
+# install basic packages
+RUN apt-get update && apt-get install -y \
+    curl \
+    wget \
+    git \
+    vim \
+    nano \
+    unzip \
+    zip \
+    p7zip-full \
+    python3 \
+    python3-pip \
+    python3-venv \
+    python3-dev \
+    python-is-python3 \
+    build-essential \
+    openssh-server \
+    tmux \
+    gettext \
+    sudo \
+    ffmpeg \
+    libsm6 \
+    libxext6 \
+    && pip install jupyter \
+    && rm -rf /var/lib/apt/lists/* # removes cache
+
+RUN pip install virtualenv \
+    && wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh \
+    && bash /tmp/miniconda.sh -b -p /opt/conda \
+    && rm /tmp/miniconda.sh \
+    && /opt/conda/bin/conda init
```

Comment:
Just a thing here, not necessarily for today, but IMHO for the PR overall: we don't use Anaconda's repos and channels anywhere in the codebase, afaik. We have replaced this with miniforge and micromamba ([example](https://github.com/All-Hands-AI/OpenHands/blob/39dad706ca43c4f3e8c8e90134e5c20f9f65d550/openhands/runtime/utils/runtime_templates/Dockerfile.j2#L107)). These two are compatible with the other conda, and we can set the channel to community repositories.

The reason is Anaconda's weird licensing. It's [not open source](https://github.com/All-Hands-AI/OpenHands/issues/1274), and while it doesn't have unexpected terms for individuals or academia (iirc), it _does_ have for companies/employees of 200 people or more. Please note also, the `-b` parameter of the script runs it silently, which, according to Anaconda, means that the user is "assumed to have agreed". (I think, if we must for some reason use this, we'd need to notify people in some very clear way.)